src/Engine/Renderer/Material/BlinnPhongMaterial.hpp

```diff
@@ -87,6 +80,8 @@ class RA_ENGINE_API BlinnPhongMaterial final : public Material {
     Core::Utils::Color m_ks{0.0, 0.0, 0.0, 1.0};
     Scalar m_ns{1.0};
     Scalar m_alpha{1.0};
+    bool m_hasPerVertexKd{false};
+    bool m_renderAsSplat{false};
```

Comment:
I'm pretty sure this (`m_renderAsSplat`) has nothing to do here and must be managed elsewhere.
About `m_hasPerVertexKd` I'm not sure